.bazelrc

```diff
@@ -34,8 +34,7 @@ build --symlink_prefix=dist/
 build --nowatchfs
 
 # Turn off legacy external runfiles
-run --nolegacy_external_runfiles
-test --nolegacy_external_runfiles
+build --nolegacy_external_runfiles
```

Comment:
Only doing this for run/test meant the analysis cache was broken after not doing it for build.

Reply:
Should it just be set for all of the three commands? run/test & build?

Reply:
run + test inherit from build, so this change sit to be set 100% of the time

See "Option precedence:" in https://bazel.build/run/bazelrc

Reply:
Ohh, I didn't know that and always specified explicitly. Nice finding